### DISCUSSION TITLE
CLSM error: Albedo NIR factor file not specified

### DISCUSSION THREAD

**Discussion by cbachand** [NASA-LIS/LISF#1016] [Q&A] [answered]
Hi David, 

I am trying to run CLSM over a small basin in High Mountain Asia at 1 km resolution. I hope to use LIS to generate an ensemble of snow and other hydrologic variables (runoff, etc.) to use for a snow data assimilation project I am working on. 

However when I perform parameter processing with LDT, I get the error: "Albedo NIR factor file not specified". I am not sure why I am getting this error, since I have specified an Albedo NIR factor file in my config file. 

Also, do my input specifications look alright? I based these off of the CLSM test case. I am used to running GEOSldas, so I haven't had to manually set all of these parameters before. 

Here is my ldtlog and ldt.config files: 

[ldtlog.txt](https://github.com/NASA-LIS/LISF/files/8256754/ldtlog.txt)
[ldt_clsm_processing.txt](https://github.com/NASA-LIS/LISF/files/8256759/ldt_clsm_processing.txt)

Thank you so much for your help!

**Comment by dmocko** [comment, marked as answer]
I believe that you are seeing this error because your path to the Albedo NIR factor file is more than 100 characters.  In the code, the character string for this config entry is assumed to be less than 100 characters.  Please use a symbolic link in your running directory to the data directory, and then edit your ldt.config file to point to the data using the symbolic link.

Also, you are using global data at 0.25-degree for your 1-km domain, so you will not get realistic parameters in your running domain.  If you want to run at 1-km, you will need to first pre-process (create) CLSM parameters at 1-km for that domain.

**Reply by cbachand** [reply]
Hi David, 

Thank you, that is very helpful. Could you point me towards documentation or example config files that could help me pre-process CLSM parameters at 1-km resolution? 

Thank you!

**Reply by dmocko** [reply]
Hello,

I should have been more clear; sorry for the confusion.  The LIS team will need to pre-process CLSM parameters on Discover, as the required codes and source datasets for the pre-processing are currently not able to be shared.  Please see the README on Discover as well as on the LIS Data Portal:

https://portal.nccs.nasa.gov/lisdata_pub/data/PARAMETERS/clsm25_parms/README

Can you please let me know more about your intended simulation?  What is the project or proposal, and are you working with anyone on the LIS team?  I see that you are already on Discover.

David

**Comment by cbachand** [comment]
Hi David,

Thank you for the clarification. I am new to LIS, so I am still learning a
lot. I am trying to assimilate Sentinel-1 snow depth observations into CLSM
using a particle batch smoother, which I am implementing myself. To do
this, I need to generate an ensemble (~100 members) of snow depth and other
hydrologic variables (SWE, runoff, etc.). I hope to run the model from
October 2016 - September 2017.  My study domain is in a mountainous area
with complex topography, so I am hoping to run CLSM at high resolution (1
km or 5 km).

My study domain is roughly bounded by these coordinates (I had lat/lon
mixed up in the original .config file I shared):

Lower left lon: 70.5
Lower left lat: 34.5
Upper right lon: 73.5
Upper right lat: 37

This domain is in the Indus basin along the border of Afghanistan and
Pakistan.

Originally, I was trying to generate this ensemble using GEOSldas, but I
cannot output the entire ensemble with GEOSldas, so I have decided to
switch to LIS. I am a student working with Dr. Manuela Girotto at UC
Berkeley, and I am not working directly with anyone on the LIS team.

Thank you!
Claire

**Reply by cbachand** [reply]
I also plan to use MERRA2 meteorological forcings
